src/components/v5/common/ActionSidebar/partials/Motions/steps/OutcomeStep/partials/VoteStatus/VoteStatus.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { FC, PropsWithChildren } from 'react';
+import clsx from 'clsx';
+
+import ProgressBar from '~v5/shared/ProgressBar';
+import Icon from '~shared/Icon';
+import { VoteStatusProps } from './types';
+import { MotionVote } from '~utils/colonyMotions';
+
+const displayName =
+  'v5.common.ActionSidebar.partials.motions.Motion.steps.OutcomeStep.partials.VoteStatus';
+
+const VoteStatus: FC<PropsWithChildren<VoteStatusProps>> = ({
```

Comment:
Is this component used anywhere else, but without avatars? If not, then I think that you can add it here directly. In other case, it should be created as a reusable component and moved to shared